lib/europeana/blacklight/api_repository.rb

```diff
@@ -100,6 +100,24 @@ def auth_params(params = {})
           wskey: blacklight_config.connection_config[:europeana_api_key]
         }.merge(params)
       end
+
+      def cache
+        @cache ||= begin
+          blacklight_config.europeana_api_cache || ActiveSupport::Cache::NullStore.new
+        end
+      end
+
+      def cache_expires_in
+        @expires_in ||= begin
+          blacklight_config.europeana_api_cache_expires_in || 24.hours
+        end
+      end
+
+      def cached(key, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.